lib/shared/util.ts

```diff
@@ -11,4 +16,72 @@ export default class Util {
     static indexOf = indexOf;
     static sortBy = sortBy;
     static format = nodeUtil.format;
+    static request = request;
+    static path = path;
+    static fs = fs;
+    static childProcess = childProcess;
+
+    static getPackageJsonFromSource(url: string) {
+        let content: any;
+
+        return this.request({
+            method: "GET",
+            uri: url,
+            json: true,
+            resolveWithFullResponse: true,
+            headers: this.defaultHeaders
+        })
+            .then((response: any) => {
+                content = response.body;
+                if (content.hasOwnProperty("templateType")) {
+                    return content;
+                }
+            })
+            .catch((error: any) => {
+                return {
+                    message: "Error retrieving package.json from src " + url,
+                    err: error
+                };
+            });
+    }
+
+    static pageExists(location: string, pageName: string) {
```

Comment:
Is this method really shared/util? Maybe move it to the page service if we do not expect to call it from outside the service itself.

Reply:
Again, in my opinion this should be handled by Fusion!

Reply:
Let's discuss this in the afternoon meeting.